src/WorkerIndexing/BindingInformation.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
```

Comment:
Please add the standard copyright/license header to all the new files - just copy it from other files.

Reply:
Done